python_on_whales/components/compose/cli_wrapper.py

```diff
@@ -354,16 +353,11 @@ def ls(
         full_cmd = self.docker_compose_cmd + ["ls", "--format", "json"]
         full_cmd.add_flag("--all", all)
         full_cmd.add_args_list("--filter", format_dict_for_cli(filters))
-
         return [
             ComposeProject(
                 name=proj["Name"],
-                created=parse_ls_status_count(proj["Status"], "created"),
-                running=parse_ls_status_count(proj["Status"], "running"),
-                restarting=parse_ls_status_count(proj["Status"], "restarting"),
-                exited=parse_ls_status_count(proj["Status"], "exited"),
-                paused=parse_ls_status_count(proj["Status"], "paused"),
-                dead=parse_ls_status_count(proj["Status"], "dead"),
+                status=proj["Status"].split("(")[0],
```

Comment:
I believe there was an issue with the rebase. I advise using merge when in doubt :)